tests/integration/test_leases.py

```diff
@@ -118,8 +118,8 @@ def test_authorized_request_with_invalid_lease_id(self, auth_headers):
         num_leases = len(LeaseModel.query.all())
         response = self.client.delete(f'{self.endpoint}/504', headers=auth_headers["pm"])
 
-        assert is_valid(response, 200)
-        assert response.json == {'Message': 'Lease Removed from Database'}
+        assert is_valid(response, 422) 
```

Comment:
I know I said to use 422 here, however it looks like we are already using 404 throughout the app, and I believe it's best to stay consistent. In regards to the 404 status the Mozilla docs say: 
> In an API, this can also mean that the endpoint is valid but the resource itself does not exist

So I guess it is not terribly incorrect to use a 404 here. What do you think?

Reply:
Using 404 in this context makes sense.